store/views.py

```diff
@@ -77,6 +95,36 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    if request.method == "POST":
+        try:
+            bid = request.POST['bid']
+            book = BookCopy.objects.get(pk=bid)
+            book.status = True
+            book.borrower = None
+            book.borrow_date = None
+            book.save()
+            return JsonResponse( {"message":"Book successfully returned."} )
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.